### PR TITLE
Feat: Staff Management

### PR DESCRIPTION
**Pull request by padishahMinecraftGamer**
….com>

### CODE REVIEW

**Review by ShaanCoding — APPROVED** [review]
Overall LGTM, looks pretty good. Awesome job for your first time using the T3 stack. Will merge in a bit, @sebasptsch mind having a look over as well, since we'll have like 4 features merged soon.

**Review by sebasptsch — DISMISSED** [review]
Just one small thing

File: packages/frontend/src/features/StaffManagement/hooks/useZocForm.ts
[the remark this answers is not in the document]
Comment (sebasptsch) [reply packages/frontend/src/features/StaffManagement/hooks/useZocForm.ts:1]: you could probably just delete this file

[the remark this answers is not in the document]
Comment (sebasptsch) [reply packages/frontend/src/features/StaffManagement/hooks/useZocForm.ts:1]: @padishahMinecraftGamer

**Comment by ShaanCoding** [comment]
Double check frontend route change in second commit @sebasptsch (cant see full code as on phone) but overall this one should be able to be merged easily.

**Review by sebasptsch — APPROVED** [review]

**Pull request merged**